### ISSUE TITLE
Ruby 3.x compatibility

### ISSUE THREAD

**Issue by n-rodriguez** [caxlsx/caxlsx#73]
Ruby 3 is out and casxlsx is not compatible with it :

it fallbacks to `casxlsx 3.0.0` :

`Using caxlsx 3.0.0 (was 3.0.3)`

```diff
-    caxlsx (3.0.3)
+    caxlsx (3.0.0)
       htmlentities (~> 4.3, >= 4.3.4)
       mimemagic (~> 0.3)
-      nokogiri (~> 1.10, >= 1.10.4)
-      rubyzip (>= 1.3.0, < 3)
+      nokogiri (~> 1.8, >= 1.8.2)
+      rubyzip (~> 1.2, >= 1.2.1)
```

**Comment by rgaufman** [comment]
I'm getting this trying to bundle:

```
caxlsx-3.0.3 requires ruby version ~> 2.3, which is incompatible with the current version, ruby 3.0.0p0
```

**Comment by rgaufman** [comment]
Same thing happens from git:

```
gem 'caxlsx', github: 'caxlsx/caxlsx'
```

```
$ bundle
Fetching https://github.com/caxlsx/caxlsx.git
caxlsx-3.0.3 requires ruby version ~> 2.3, which is incompatible with the current version, ruby 3.0.0p0
```

Any solution?

**Comment by Laykou** [comment]
you need to specify branch I believe, like `branch: :master` ?

**Comment by n-rodriguez** [comment]
> you need to specify branch I believe, like `branch: :master` ?

nope, it's master branch by default.

The only fix is fork and patch https://github.com/caxlsx/caxlsx/blob/master/axlsx.gemspec#L27

**Comment by n-rodriguez** [comment]
Or use this repo : https://github.com/walski/caxlsx/tree/ruby-3-compatibility

**Comment by Laykou** [comment]
Some work has been done here: https://github.com/caxlsx/caxlsx/pull/72 and it seems to be ready for merge soon :)

**Comment by straydogstudio** [comment]
Fixed in #72 and released in 3.0.4.

**Comment by n-rodriguez** [comment]
> Fixed in #72 and released in 3.0.4.

thank you!

**Comment by straydogstudio** [comment]
You're welcome. And special thanks to @walski who did all the work getting Travis to agree. 👍
